<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapDataTool</class>
 <widget class="QWidget" name="MapDataTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Data Tools</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Button_MapData_New">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>New Map Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_MapData_Open">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Open Map Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_MapData_Save">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Save Map Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_MapData_Export">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Export Finished Map Files</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="Button_MapTools_ReturnToMain">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Back To Main Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Editors</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_MapTool_CityEditor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>City Editor</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/.png/Block of flats_#222222_64px.png</normaloff>:/.png/Block of flats_#222222_64px.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_MapTool_TurnEventsEditor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Turn Events Editor</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/.png/Calendar with circles_#222222_64px.png</normaloff>:/.png/Calendar with circles_#222222_64px.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_MapTool_AIEditor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>AI Editor</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/.png/chess-horse-icon.png</normaloff>:/.png/chess-horse-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Map Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LineEdit_MapTools_MapName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Map Version:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="SpinBox_MapTools_MapVersionNumber">
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Author Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Author Link:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Map Type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LineEdit_MapTools_Author"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="LineEdit_MapTools_AuthorLink"/>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="Radio_MapTools_DotCity">
         <property name="text">
          <string>Dot City</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Radio_MapTools_DotScene">
         <property name="text">
          <string>Dot Scene</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Map Art Zip Location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="LineEdit_MapTools_MapArtZipLocation"/>
       </item>
       <item>
        <widget class="QToolButton" name="ToolButton_MapTools_MapArtZipLocationFinder">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Map Art Loading:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="Button_MapTools_EditMapLoadingYears">
       <property name="text">
        <string>Edit Map Artwork Loading Years</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>City Scipts Folder:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="LineEdit_MapTools_CityScriptFolder"/>
       </item>
       <item>
        <widget class="QToolButton" name="ToolButton_MapTools_CityScriptFolderFinder">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>City Script Loading:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="Button_MapTools_EditCityLoadingYears">
       <property name="text">
        <string>Edit City Script Loading Years</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>City Art Zip File:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Turn Events XML File:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Newspaper Zip File:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLineEdit" name="LineEdit_MapTools_CityArtZipLocation"/>
       </item>
       <item>
        <widget class="QToolButton" name="Button_MapTools_CityArtZipLocationFinder">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLineEdit" name="LineEdit_MapTools_TurnEventsXMLFileLocation"/>
       </item>
       <item>
        <widget class="QToolButton" name="Button_MapTools_TurnEventsXMLFileLocationFinder">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLineEdit" name="LineEdit_MapTools_NewspaperZipFileLocation"/>
       </item>
       <item>
        <widget class="QToolButton" name="LineEdit_MapTools_NewspaperZipFileLocationFinder">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>AI Script Folder:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>AI Logo Zip File:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>AI Starting Options:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="Button_MapTools_EditStartingAIFiles">
       <property name="text">
        <string>Edit AI Starting Options</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="LineEdit_MapTools_AIScriptFolder"/>
       </item>
       <item>
        <widget class="QToolButton" name="Button_MapTools_AIScriptFolderFinder">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLineEdit" name="LineEdit_MapTools_AiLogoZipLocation"/>
       </item>
       <item>
        <widget class="QToolButton" name="Button_MapTools_AiLogoZipLocationFinder">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Factory Size Adjust:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Branch Dealer Adjust</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QDoubleSpinBox" name="SpinBox_MapTools_BranchDealerAdjust">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QDoubleSpinBox" name="SpinBox_MapTools_FactorySizeAdjust">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="GroupBox_MapTools_AIStartingOptions">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>AI Starting Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="Table_MapTools_StartingAINumbersAndFiles">
          <column>
           <property name="text">
            <string>Number Of AI</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>AI File</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Random Number of AI (-1 For All):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SpinBox_MapTools_StartingAINumberSetter">
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>AI Starting File:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineEdit_MapTools_StartingAIFileAdder"/>
          </item>
          <item>
           <widget class="QToolButton" name="Button_MapTools_AddStartingAINumbersAndFile">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="Button_MapTools_RemoveStartingAINumberAndFile">
            <property name="text">
             <string>–</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <widget class="QGroupBox" name="Group_MapTools_MapArtwork_Change">
   <property name="geometry">
    <rect>
     <x>577</x>
     <y>67</y>
     <width>561</width>
     <height>644</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Map Art Loading Years</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="Table_MapTools_MapArtLoadingYears">
        <column>
         <property name="text">
          <string>Year</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Map File</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Map Year:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SpinBox_MapTools_MapArtworkLoadingYear">
          <property name="minimum">
           <number>1880</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>1900</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Map File Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LineEdit_MapTools_MapArtworkLoaderFileName"/>
        </item>
        <item>
         <widget class="QToolButton" name="Button_MapTools_AddMapArtworkToLoader">
          <property name="text">
           <string>+</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="Button_MapTools_RemoveMapArtworkToLoader">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="GroupBox_MapTools_CityScriptLoadingYear">
   <property name="geometry">
    <rect>
     <x>577</x>
     <y>67</y>
     <width>561</width>
     <height>644</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>City Script Loading Years</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="Table_MapTools_CityScriptLoadingYears">
        <column>
         <property name="text">
          <string>Year</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>City Script</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Year:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SpinBox_MapTools_CityLoadingYear">
          <property name="minimum">
           <number>1900</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>City Script:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LineEdit_MapTools_CityScriptFileName"/>
        </item>
        <item>
         <widget class="QToolButton" name="Button_MapTools_AddCityScriptYearLoader">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="Button_MapTools_RemoveCityScriptYearLoader">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>GroupBox_MapTools_CityScriptLoadingYear</zorder>
  <zorder>Group_MapTools_MapArtwork_Change</zorder>
  <zorder>GroupBox_MapTools_AIStartingOptions</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_4</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_MapTools_EditMapLoadingYears</sender>
   <signal>clicked()</signal>
   <receiver>Group_MapTools_MapArtwork_Change</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_MapTools_EditStartingAIFiles</sender>
   <signal>clicked()</signal>
   <receiver>GroupBox_MapTools_AIStartingOptions</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_MapTools_EditCityLoadingYears</sender>
   <signal>clicked()</signal>
   <receiver>GroupBox_MapTools_CityScriptLoadingYear</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
